### ISSUE TITLE
P3: Remove "File/Link Uploads" Scope Tab

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/connect-app#3305]
**Objective:** Remove the file/link upload capability currently displayed at the bottom of the Scope tab once the intake form is submitted. The objective is streamline areas where file and link sharing within the platform. As the Scope Tab will eventually be removed and scope display/interactions will move to the scope card in project plan, sharing files/links in this section of the platform are no longer required.

<p></p>

**Requirements:**

- Remove file/link upload option at the bottom of the Scope tab in projects

**Comment by ajefts** [comment]
Link to feature: https://tcprod.productboard.com/feature-board/planning/features/2756963

**Comment by vikasrohit** [comment]
@maxceem I would remove the file sub section from the forms, however, I am afraid if removing that would remove the capability of the users to upload the files to the system completely. I guess, we used to use presence of the file subsection to enable the file upload from the left panel, I am not sure if we carried that logic to the new `Assets Library` page as well. Do you have idea about it?

**Comment by maxceem** [comment]
@vikasrohit yes, we keep that logic. Now we always show the menu item, but we hide `Add new...`. button on the `Assets Library` page, ref:
- https://github.com/appirio-tech/connect-app/blob/dev/src/projects/detail/containers/AssetsInfoContainer.jsx#L575-L578
- https://github.com/appirio-tech/connect-app/blob/dev/src/projects/detail/containers/AssetsInfoContainer.jsx#L448-L455

**Comment by maxceem** [comment]
> I would remove the file sub section from the forms

I guess if you remove it from the forms, it wouldn't be shown during project creation also. As from the issue description, it looks like we still want to keep it the intake form:
> Remove the file/link upload capability currently displayed at the bottom of the Scope tab once the intake form is submitted

**Comment by vikasrohit** [comment]
> @vikasrohit yes, we keep that logic. 

Good to know that. We need to get rid of that logic now to implement this feature. Only concern I have now is about the original reason for which we added that logic in first place and that was because we wanted to hide file upload feature for certain customers which we achieved by removing the file upload sub section from their project templates.

> I guess if you remove it from the forms, it wouldn't be shown during project creation also. As from the issue description, it looks like we still want to keep it the intake form:

We don't yet have any form on the production which has the file upload visible during project creation though we have that feature/logic in place now to have file upload in project creation form.
